src/main/java/picard/sam/PositionBasedDownsampleSam.java

```diff
@@ -55,41 +45,112 @@
+              "size (when running mark-duplicates) THe downsampling method done keeps (physically) close reads together, so that (except " +
+              "for the reads near the boundary of the circle) optical duplicates are kept or dropped as a group. " +
+              "By default the program expects the read names to have 5 or 7 fields separated by colons (:) and it takes the last two " +
+              "to indicate the x and y coordinates of the reads within the \"tile\" from whence it was sequenced. See " +
+              "DEFAULT_READ_NAME_REGEX for more detail. The program traverses the INPUT twice. First " +
+              "to find out the size of each of the tiles, and next to perform the downsampling. " +
+              "Downsampling invalidates the duplicate flag (as reads marked as duplicates prior to downsampling might not be duplicates " +
+              "afterwards.) Thus, the default setting also removes the duplicate information. " +
+              "\n\n" +
+              "Example\n\n" +
+              "java -jar picard.jar PositionBasedDownsampleSam \\\n" +
+              "      I=input.bam \\\n" +
+              "      O=downsampled.bam \\\n" +
+              "      FRACTION=0.1\n" +
+              "\n" +
+
+              "Caveats\n\n" +
+              "Note 1: " +
+              "This method is <b>technology and read-name dependent</b>. If the read-names do not have coordinate information " +
+              "embedded in them, or if your BAM contains reads from multiple technologies (flowcell versions, sequencing machines) " +
+              "this will not work properly. It has been designed to work with Illumina technology and reads-names. Consider " +
+              "modifying {@link #READ_NAME_REGEX} in other cases. " +
+              "\n\n" +
+              "Note 2: " +
+              "The code has been designed to simulate, as accurately as possible, sequencing less, <b>not</b> for getting an exact " +
+              "downsampled fraction (Use {@link DownsampleSam} for that.) In particular, since the reads may be distributed non-evenly " +
+              "within the lanes/tiles, the resulting downsampling percentage will not be accurately determined by the input argument " +
+              "FRACTION. " +
+              "\n\n"+
+              "Note 3:" +
+              "Consider running {@link MarkDuplicates} after downsampling in order to \"expose\" the duplicates whose representative has " +
+              "been downsampled away." +
+              "\n\n" +
+              "Note 4:" +
+              "Due to the way the downsampling it performed, it assumes a uniform distribution of reads in the flowcell. This is violated " +
+              "if one uses an input that was already downsampled with PositionBasedDownsampleSam. To guard against this, a " +
+              "PG record is placed in the header ouf the output and if the input is found to contain this record, the program will " +
+              "abort.",
```

Comment:
Too passive and thisy.  How about?
```
"Note 4: " +
"The downsampling assumes a uniform distribution of reads in the flowcell. " +
"Input already downsampled with PositionBasedDownsampleSam violates this assumption. " +
"To guard against such input, " +
"PositionBasedDownsampleSam always places a PG record " +
"in the header of its output, " +
"and aborts whenever it finds a PG record in its input."
```
There are other improvements, but Note 4 needs the most help.

And notice that if you break the strings up into self-contained clauses,
the resulting code is much easier for human eyes to parse and proof.